fuzz/fuzz.c

```diff
@@ -0,0 +1,18 @@
+#include "flow.h"
+#include "he_internal.h"
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
```

Comment:
What do you think about renaming this function to something like `FuzzerTestReadPacket`? Technically this fuzzing harness could be used with other fuzzers too, not just the LLVM one! 

This function is possible one of many functions that help us pass fuzzing inputs to different parts of lightway-core, so it would be nice to state that this fuzzes `ReadPacket`.

Reply:
`LLVMFuzzerTestOneInput` is the name the generate main function will attempt to link to. So we need a `LLVMFuzzerTestOneInput` no matter what.

The question then becomes do we write a new executable per fuzz target OR do we share it all in a single fuzz binary.

Reply:
Sorry, that was me not understanding https://llvm.org/docs/LibFuzzer.html's docs on `LLVMFuzzerTestOneInput`.  It wasn't super obvious from docs there that this name was special. 

Re: how to handle fuzz targets: doesn't seem like there's any superior way.  This project uses #ifdefs: http://docs.frrouting.org/projects/dev-guide/en/latest/fuzzing.html#targets